app.rb

```diff
@@ -1,9 +1,33 @@
 require 'sinatra/base'
+require_relative './lib/peeps'
 
 class Chitter < Sinatra::Base
+  enable :sessions
+
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    @timeline = Peeps
+    erb :index
+  end
+
+  post '/' do
+    Peeps.add(peep: params[:peep])
+    redirect '/'
+  end
+
+  post '/search' do
```

Comment:
If you like, you can show search results in the `get '/'` by doing something like this

```
get '/' do
  if params[:search]
    @peeps = Peeps.all.select { |peep| peep.peep.include? params[:search] }
  else
    @peeps = Peeps.all
  end
end
```